devel/howdoi/Portfile

```diff
@@ -4,33 +4,24 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                howdoi
-version             1.1.9
+version             2.0.19
 categories          devel python
-platforms           darwin
 license             MIT
 supported_archs     noarch
-
-python.default_version 27
-
 maintainers         nomaintainer
 
 description         instant coding answers via the command line
 long_description    {*}${description}
 homepage            https://pypi.python.org/pypi/howdoi
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
-
-distname            ${python.rootname}-${version}
-
-checksums           rmd160  9c1fdc3dfd03fd717b4978b05928bc0d565f6f10 \
-                    sha256  2b4a0ebc39c9e0489a17ef73e3eec7dd4a2975e07b995406887c639481da2bb5
 
-depends_build-append \
-    port:py${python.version}-setuptools
+checksums           rmd160  bb99d6f9638be1ea3e3e51bca244e1c66ef76164 \
+                    sha256  4baa5270a98e9d001094d873a5a1b6a32eac021009cfc40cd9aeede633cb7556 \
+                    size    27109
 
-depends_lib-append \
-    port:py${python.version}-requests \
-    port:py${python.version}-requests-cache \
-    port:py${python.version}-pygments \
-    port:py${python.version}-pyquery
+python.default_version 310
 
-livecheck.type      pypi
+depends_build-append   port:py${python.version}-setuptools
+depends_lib-append     port:py${python.version}-requests \
+                       port:py${python.version}-requests-cache \
+                       port:py${python.version}-pygments \
+                       port:py${python.version}-pyquery
```

Comment:
the whitespacing here ins not correct, it should be multiples of four - your `port:py${python.version}-*` starts at 24. If you have a modeline in your Portfile you can just hit "tab" and it will do the alignment correctly.

I would suggest:
```
depends_build-append \
                    port:py${python.version}-setuptools

depends_lib-append  port:py${python.version}-requests \
                    port:py${python.version}-requests-cache \
                    port:py${python.version}-pygments \
                    port:py${python.version}-pyquery
```

Reply:
also, you're missing quite some dependencies; please check [upstream's setup.py](https://github.com/gleitz/howdoi/blob/38435e64680a7422e6937b8e44973bee65ff7cb2/setup.py#L96) file and add all of them.

Reply:
once you have all of them you can add a test-phase as well:
```
test.run            yes
test.cmd            ${python.bin} -m unittest
test.target
test.env            PYTHONPATH=${worksrcpath}/build/lib
```

and can check to make sure it passes with Python 3.10

Reply:
@reneeotten Thank you for help! Will fix these tonight.

Reply:
> the whitespacing here ins not correct, it should be multiples of four - your `port:py${python.version}-*` starts at 24. If you have a modeline in your Portfile you can just hit "tab" and it will do the alignment correctly.
> 
> I would suggest:
> 
> ```
> depends_build-append \
>                     port:py${python.version}-setuptools
> 
> depends_lib-append  port:py${python.version}-requests \
>                     port:py${python.version}-requests-cache \
>                     port:py${python.version}-pygments \
>                     port:py${python.version}-pyquery
> ```

If I use tab, it inputs tab (BBEdit).
I ran `port lint --nitpick howdoi` like you advised, no errors or warnings. Should I still change alignment?

Reply:
"port lint" does not check for alignment, but yes you should do it correctly